### DISCUSSION TITLE
Only one pptp client can get an IPv6 address

### DISCUSSION THREAD

**Discussion by sdasdasdwsaaa** [accel-ppp/accel-ppp#214] [General]
I used accel-ppp to build a pptp server, and also configured dhcpv6 related configurations, and found that only the first connected client can get the ipv6 address assigned by dhcp, and the other clients can only get the address beginning with fe80

accel-cmd -V
accel-cmd 1.12.0-266-gfd46f0d

**Comment by sdasdasdwsaaa** [comment]
operating system ：Ubuntu 22.04.3 LTS (GNU/Linux 5.15.0-86-generic x86_64)

**Comment by DmitriyEshenko** [comment]
Hi @sdasdasdwsaaa , it looks like you have not properly configured ipv6 pool. Try to check this section https://github.com/accel-ppp/accel-ppp/blob/master/accel-pppd/accel-ppp.conf#L316-L326
